src/protocol_nmea.cpp

```diff
@@ -54,6 +50,32 @@ int ssnprintf(char buf[], int bufSize, const char *fmt, ...)
     return l;
 }
 
+/**
+ * Sets Gsv filter. 
+ * 
+ * @param seedArr - filters for each constellation (uint8_t seedArr[SAT_SV_GNSS_ID_COUNT])
+ */
+void nmea_setGsvFilter(uint8_t* seed)
+{
+    for (int i = 0; i < SAT_SV_GNSS_ID_COUNT && seed != nullptr; i++)
+    {
+        s_gsvMask.constMask[i] = seed[i];
+    }
+}
+
+/**
+ * Gets Gsv filter. 
+ * 
+ * @param seedArr - filters for each constellation (uint8_t seedArr[SAT_SV_GNSS_ID_COUNT])
+ */
+void nmea_getGsvFilter(uint8_t* seed)
```

Comment:
Should be renamed to "nmea_plantGsvFilter()".  :grinning: :seedling:

Reply:
if it wasn't in the SDK in half tempted to do it